File: POO/src/poo/Gerente.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poo;

/**
 *
 * @author Icaro
 */
public class Gerente extends Funcionario{
    @Override
    public double calcularParticipacaoNosLucros(){
    return getSalario() * 0.5;
    }
}

File: POOC/POOC/br.com.funcionario/Gerente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Func
{
    class Gerente : Funcionario
    {
        public Gerente(String a) { 
        
        }
        public override double calcularParticipacaoNosLucros() {

            return getSalario() * 0.5; 
        }
    }
}
